app/serializers/reviewable_note_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ReviewableNoteSerializer < ApplicationSerializer
+  attributes :id, :content, :created_at, :updated_at
+
+  has_one :user, serializer: BasicUserSerializer, embed: :objects
```

Comment:
Since this is an association, we need to ensure that all controller actions using this serializer is eager loading the `user` association for `reviewable_notes`.

Reply:
Looks like this is where the preloading is done:

https://github.com/discourse/discourse/blob/f0e72f620618147a292cf2b5740c5535e921dd68/app/models/reviewable.rb#L429-L438

Reply:
Thanks for the pointer, fixed. 🙂